src/collections/_documentation/platforms/php/index.md

```diff
@@ -345,16 +356,50 @@ The best adapter available is automatically selected when creating a client inst
 through the client builder, but you can override it using the appropriate methods.
 
 ```php
+use Http\Discovery\HttpAsyncClientDiscovery;
+use Http\Discovery\MessageFactoryDiscovery;
+use Http\Discovery\StreamFactoryDiscovery;
+use Http\Discovery\UriFactoryDiscovery;
+use Http\Message\ResponseFactory;
+use Http\Message\UriFactory;
 use Sentry\ClientBuilder;
+use Sentry\HttpClient\HttpClientFactory;
+use Sentry\HttpClient\HttpClientFactoryInterface;
 use Sentry\Transport\HttpTransport;
 use Sentry\State\Hub;
+use Sentry\Transport\TransportFactoryInterface;
 
 $options = ['dsn' => '___PUBLIC_DSN___'];
 
-$transport = new HttpTransport($options, HttpAsyncClientDiscovery::find(), MessageFactoryDiscovery::find());
+$httpClientFactory = new HttpClientFactory(
+    UriFactoryDiscovery::find(),
+    MessageFactoryDiscovery::find(),
+    StreamFactoryDiscovery::find(),
+    null,
+    'sentry.php',
+    '2.3'
+);
+
+$transportFactory = new class($httpClientFactory) implements TransportFactoryInterface  {
```

Comment:
Instead of creating a custom factory, the `DefaultTransportFactory` can be used too in this case as the logic is basically the same. Of course if there was custom logic involved creating your own transport factory is still needed